lib/utils/authorizeSilently.ts

```diff
@@ -29,7 +29,7 @@ export const authorizeSilently = (
 
       cleanup();
 
-      if (event.data && !event.data.error) {
+      if (event.data && event.data.response) {
```

Comment:
nit: could this now be as well look like this?
```typescript
      if (event.data?.response) {
        const { code } = event.data.response;
        return resolve(code);
      } else {
        const error = event.data?.error ??
          new RenewTokenError({
            message: "Invalid message received from server"
          });
        return reject(error);
      }
```

Same for `if (iframe && iframe.parentNode)` etc.

Reply:
True, still not used to go for these optional chainings as my go to 🙂

Reply:
Changed here: 7e5c5451b494a2be8af8a355ed7a8459c0c560cd 👍